java/client/src/main/java/glide/api/models/Transaction.java

```diff
@@ -27,4 +30,18 @@ public class Transaction extends BaseTransaction<Transaction> {
     protected Transaction getThis() {
         return this;
     }
+
+    /**
+     * Change the currently selected Redis database.
+     *
+     * @see <a href="https://redis.io/commands/select/">redis.io</a> for details.
+     * @param index The index of the database to select.
+     * @returns Command Response - A simple <code>OK</code> response.
+     */
+    public Transaction select(long index) {
+        RedisRequestOuterClass.Command.ArgsArray commandArgs = buildArgs(Long.toString(index));
+
+        protobufTransaction.addCommands(buildCommand(Select, commandArgs));
+        return getThis();
```

Comment:
Could be simple
```suggestion
        return this;
```
because `getThis` is needed by the superclass